jax/experimental/jax2tf/jax2tf.py

```diff
@@ -852,7 +852,18 @@ def _unexpected_primitive(p: core.Primitive, *args, **kwargs):
 tf_impl[xla.device_put_p] = lambda x, device=None: x
 
 tf_impl[lax.neg_p] = tf.math.negative
-tf_impl[lax.sign_p] = tf.math.sign
+
+def _sign(x):
```

Comment:
I think that for now we should avoid adding more casts. We need to decide how to handle more systematically the unimplemented dtypes